examples/tutorials/low_level_tutorial.py

```diff
@@ -126,17 +120,16 @@
 ## Optimizers and Integrators
 
 All optimizers and integrators share a similar interface. They accept a model and
-return two functions: `init_fn` and `update_fn`. The `init_fn` function returns the
-initialized optimizer-specific state, while the `update_fn` function updates the
+return two functions: `init_fn` and step_fn`. The `init_fn` function returns the
+initialized optimizer-specific state, while the step_fn` function updates the
```

Comment:
I think this is changed in this PR but haven't reached relevant files to be sure.